distributions/demo-server/build.gradle

```diff
@@ -40,6 +55,23 @@ dependencies {
 
 addCloudExtensionDependency(project)
 
+task createSecretsFile() {
```

Comment:
can you add a sanity check (or TODO) that this is only called for the local environment? (for prod, we shouldn't store raw secrets, we should get them from the cloud environment somewhere where they should be encrypted)

(I see you have a TODO for this in java too)

Reply:
I think this is OK as is since the demo-server is never put into production. The file is only generated for that demo-server jar and not any other jars

Reply:
Oops, I think I hit the wrong button to start a review on my comment

Reply:
Ohh ok cool, didn't realize this was in the demo server. Thx